democracy_club/apps/projects/templates/projects/projects_home.html

```diff
@@ -39,6 +39,6 @@ <h3><a href="{% url "voting_information_api" %}">Election data and APIs</a></h3>
     </div>
     <h2>Year round services</h2>
     <h3><a href="{% url "projects:representatives" %}">Representatives</a></h3>
-    <p>Approximately 400 organisations with 22,000 elected people make decisions every day about everything from schools
+    <p>Approximately 300 organisations with 19,000 elected people make decisions every day about everything from schools
```

Comment:
By my count, there are 382 councils in England/Wales/Scotland/NI (that's 361 lower tier/unitary councils plus 21 counties) and a smattering of combined authorities.

Obviously all of this is about to change, but "about 400" seems like it is still right to me.

Reply:
Thanks! yes, I was accidentally only thinking about England. Leave 400.